benches/protobuf_convert_output.rs

```diff
@@ -0,0 +1,34 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use prost::Message;
+use tailcall::blueprint::GrpcMethod;
+use tailcall::grpc::protobuf::ProtobufSet;
+
+const PROTO_FILE_PATH: &str = "src/grpc/tests/proto/greetings.proto";
+const SERVICE_NAME: &str = "greetings.Greeter.SayHello";
```

Comment:
Let's create a protofile in the benches.

Reply:
done